### PR TITLE
Add vignette

### PR DESCRIPTION
**Pull request by llrs**
Following the README.Rmd I created this PR to add a vignette (related to #2).
While creating the vignette I found is a lack of functions to use the data retrieved. 
Also having a summary of the results obtained printed on the console when performing a query could come handy.

### CODE REVIEW

**Comment by sckott** [comment]
thanks for this @llrs having a look

File: vignettes/jaod.Rmd [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+Vignettes are long form documentation commonly included in packages. Because they are part of the distribution of the package, they need to be as compact as possible. The `html_vignette` output type provides a custom style sheet (and tweaks some options) to ensure that the resulting html is as small as possible. The `html_vignette` format:
```
Comment (sckott) [inline vignettes/jaod.Rmd]: Can you remove this paragraph and bulleted list?  We usually don't introduce the vignette format in vignettes.

**Review by sckott — CHANGES_REQUESTED** [review]

File: vignettes/jaod.Rmd (hunk 1 above)
Comment (llrs) [reply vignettes/jaod.Rmd]: Oh, sorry I forgot to remove it
```suggestion
```

**Comment by sckott** [comment]
Looks like you need to merge in changes from master before we can merge here

**Review by sckott — APPROVED** [review]

**Pull request merged**